package.json

```diff
@@ -45,6 +45,9 @@
 		"percy": "if [[ $(git rev-parse --symbolic-full-name --abbrev-ref HEAD) != \"master\" ]]; then percy storybook ./storybook-static; else echo \"ERROR: percy should not be run on master because these snapshots are auto-approved\"; fi;",
 		"test": "jest"
 	},
+	"bin": {
+		"pc-check-tokens": "./check-tokens/index.js"
```

Comment:
Wouldn't it be better/cleaner to put this in a directory `bin`? There might be more scripts in the future so they can be grouped together in this folder. And also it's something that is widely understood (like `dist`, `src`, `build`) so the root folder has a clean structure. And the script itself can be named `check-tokens.js`. There's no need to name it `index` right? 😊

Reply:
good idea 👍